en/components/drop_down.md

```diff
@@ -393,6 +393,110 @@ When [allowItemsFocus]({environment:angularApiUrl}/classes/igxdropdowncomponent.
+Fist define a custom palette and pass our main colors:
+```scss
+$my-primary-color: #ffffff;
+$my-secondary-color: #7344df;
+
+$my-color-palette: igx-palette(
+    $primary: $my-primary-color,
+    $secondary: $my-secondary-color
+);
+```
+
+Now we can specify some more colors to style our component. It is really convenient to take use of the previously created [palette](../themes/palette.md) and base our new colors on the primary&secondary colors defined. Lets say we are going for some different shades of purple.
+```scss
+$my-light-purple: igx-color($my-color-palette, "secondary", 100);
+$my-medium-purple: igx-color($my-color-palette, "secondary", 300);
+$my-medium-plus-purple: igx-color($my-color-palette, "secondary", 400);
+$my-dark-purple: igx-color($my-color-palette, "secondary", 600);
+);
+```
+
+In order to see our custom palette and colors applied, we need to pass these to a theme function.
+So in one bold move we will [`create a custom theme`](../themes/component-themes.md##creating-themes) and pass our cool colors to a number of predefined `igx-drop-down-theme parameters`. Let's say we have decided modifying these specific parameters will be more than sufficient to make our component look the way we like:
+```scss
+$custom-drop-down-theme: igx-drop-down-theme(
+    $background-color: $my-light-purple,
+    $item-text-color: $my-primary-color,
+    $header-text-color: $my-dark-purple,
+
+    $selected-item-background: $my-medium-plus-purple,
+    $selected-item-text-color: $my-primary-color,
+    $selected-hover-item-background: $my-medium-plus-purple,
+    $selected-hover-item-text-color: $my-primary-color,
+    $selected-focus-item-background: $my-medium-plus-purple,
+    $selected-focus-item-text-color: $my-primary-color,
+
+    $focused-item-background: $my-medium-purple,
+    $focused-item-text-color: $my-primary-color,
+
+    $hover-item-background: $my-primary-color,
+    $hover-item-text-color: $my-dark-purple
+);
+```
+As in this particular sample we are going to use a button to toggle the `igx-drop-down`, we can go a bit further and style it as well. So we go a bit outside the `igx-drop-down` topic and to complement the overall drop-down theme styling we will create a custom button theme passing it our color palette like: 
+```scss
+$custom-button-theme: igx-button-theme(
+    $palette: $my-color-palette
+);
+
+```
+
+#### Applying
+All that's left is to properly scope our newly created themes.
+
+##### Globally
+In case you want this newly created `igx-drop-down` theme to be applied [`globally`](../themes/component-themes.md##creating-themes) in your app (to all of the components of this type, keep in mind we are actually extending igx-drop-down-theme), all that is needed is to include the theme in your app root style file. The same applies for our custom button theme:
```

Comment:
```suggestion
In case you want this newly created `igx-drop-down` theme to be applied [`globally`](themes/component-themes.md#creating-themes) in your app (to all of the components of this type, keep in mind we are actually extending igx-drop-down-theme), all that is needed is to include the theme in your app root style file. The same applies for our custom button theme:
```